res/TensorFlowPythonExamples/examples/prelu/__init__.py

```diff
@@ -1,7 +1,5 @@
 import tensorflow as tf
-from tensorflow.compat.v1.keras import layers
 
-model = tf.compat.v1.keras.Sequential()
-model.add(layers.PReLU())
-# TODO Find a way to freeze Keras model for inference
-model.build((1, 1))
+lhs_ = tf.compat.v1.placeholder(dtype=tf.float32, name="input", shape=[1, 4, 4, 3])
+prelu = tf.keras.layers.PReLU(shared_axes=[1, 2])
```

Comment:
By the way, what happens if we remove ``shared_axes`` from this code?

Reply:
It still works, this just demonstrates the parameter passing.

Reply:
(Optional) It may be better to have both examples and keep this as the most simplest one (e.g. remove ``shared_axes``).